CRM/Utils/Check/Component/Source.php

```diff
@@ -17,90 +17,28 @@
 class CRM_Utils_Check_Component_Source extends CRM_Utils_Check_Component {
 
   public function getRemovedFiles() {
-    $files[] = '[civicrm.packages]/Auth/SASL';
-    $files[] = '[civicrm.packages]/Auth/SASL.php';
-    $files[] = '[civicrm.packages]/Net/SMTP.php';
-    $files[] = '[civicrm.packages]/Net/Socket.php';
-    $files[] = '[civicrm.packages]/_ORIGINAL_/Net/SMTP.php';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/Readme.md';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/license.txt';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables.min.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/css/jquery.dataTables_themeroller.css';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.dataTables.js';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.dataTables.min.js';
-    $files[] = '[civicrm.packages]/jquery/plugins/DataTables/media/js/jquery.js';
-    $files[] = '[civicrm.vendor]/pear/net_smtp/examples';
-    $files[] = '[civicrm.vendor]/pear/net_smtp/tests';
-    $files[] = '[civicrm.vendor]/pear/net_smtp/phpdoc.sh';
-    $files[] = '[civicrm.vendor]/phpoffice/phpword/samples';
```

Comment:
For someone installing from zip/tar, it's the same issue as `civicrm-core` or `civicrm-packages`. (Merely extracting won't delete things.)

On MM, you were brainstorming about whether `composer` changes the need for checks. 

In my mind, the general reason to do run-time sniff-tests on `[civicrm.vendor]` is to ensure that either [composer cleanup steps were executed](https://github.com/civicrm/civicrm-core/blob/5.69/composer.json#L109-L128) or that they're not necessary. Reasoning about what `composer` will or will not do (*across the range of configurations/environments/admins*) is not always simple. For example, just within D10, you have these two variations in process:

```bash
## Typical process for core-contributor is like...
git clone https://github.com/drupal/drupal mysite
cd mysite
composer install
```
```bash
## Typical process for site-builder is like...
git clone https://github.com/drupal/recommended-project mysite
cd mysite
composer install
```

One has a public `vendor` (*where you do should think about HTTPD policies and example files*).  The other has a private `vendor` (*where you don't need to think about HTTPD policies or whether example files are web-accessible*). For one style, it doesn't matter if you fire Civi's composer-cleanup-scripts. For the other style, it might. (But not necessarily! Depends on HTTPD!) In either case, neither of them do fire cleanup scripts on that folder (at least, by default).

Of course, maybe these are two different things: (1) checking that deleted files are deleted vs (2) checking that non-public files are non-public. If these `[civicrm.vendor]` things are covered by some other HTTP-level check, then maybe the file-level check would be somewhat redundant?

(*Except that we still have the issue where unzipping/untarring a new version of Civi can leave you with old files...*)

Reply:
@totten I'm not sure based on this comment if a change to this PR is needed?